### ISSUE TITLE
Gallery/demo app

### ISSUE THREAD

**Issue by Leon99** [amwx/FluentAvalonia#398]
Given the inevitable comparison with WinUI 3, would be great to have a FluentAvalonia port of https://github.com/microsoft/WinUI-Gallery so it's possible to compare them side-by-side.

**Comment by amwx** [comment]
`FluentAvaloniaSamples` was the old demo app, which is now replaced by `FAControlsGallery`. Having an exact replica of the WinUI gallery isn't going to happen:

1) I don't maintain the full suite of controls for Avalonia. For the "basic controls", I just apply the Fluent v2 styling to them. Having a full demo app for those might give the impression I do and lead to bug reports here that should be directed upstream. This is why these controls are grouped together and samples are very minimal in nature in my demo app - it's just to show off the styling.
2) Even though the WinUI Gallery is MIT license, it doesn't feel right to me making a carbon copy of the app for this repo. Also, my own demo is going to reflect my own design choices and allow me to show off what Avalonia can do that may or may not be possible in WinUI, or done differently. There's also some improvements I've made to controls like `Frame` not found in WinUI that this serves as a nice "how to" demo.

There is a lot of similarity between the apps - some inspiration comes from it. Some things were ported from there (like the ControlExample controls used to differentiate examples on pages, and in the new demo app, the Design guidelines pages are straight from WinUI with just a few modifications for Avalonia). When a control is ported from WinUI to here, I usually use the sample page from WinUI to write the example page here, so the actual pages should have a similar feel. For controls that FluentAvalonia adds, I'm always open to requests (and PRs upon approval) for new examples if you feel something is missing or could be better represented. 

TLDR: I have a demo app, it has a different look and feel compared to WinUI, but it accomplishes the same thing.

**Comment by Leon99** [comment]
Thanks for the detailed elaboration @amwx, much appreciated!
I missed FAControlsGallery somehow tbh, it does look closer to what I expected. Just out of curiosity, is there a reason to keep FluentAvaloniaSamples, given it's now superseded?

**Comment by amwx** [comment]
FAControlsGallery was just merged the other day so it's brand new. I haven't gotten around to removing the old project yet
